packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -24,6 +38,148 @@ import {SparkPlugin, Page} from '@ciscospark/spark-core';
  * @name Memberships
  */
 const Memberships = SparkPlugin.extend({
+  /**
+   * Connect to the web socket to listen to incoming messages.
+   * @returns {Promise}
+   */
+  listen() {
+    // Create a common envelope that we will wrap all events in
+    this.eventEnvelope = {
+      resource: 'memberships',
+      // orgId -- TODO get from this.spark.internal.user
+      // createdBy -- TODO get from this.spark.internal.user
+      ownedBy: 'creator',
+      status: 'active',
+      created: new Date().toISOString(),
```

Comment:
Because `listen` only happens once, I would leave off `created` and add it when you handle the event. (I'll keep reading, but I'm not sure `eventEnvelope` should be a class instance.)

Reply:
As per our discussion, leaving this on to align with the webhook envelope information.